data/json/items/battery.json

```diff
@@ -63,10 +63,22 @@
     "weight": "23 g",
     "description": "A light AA battery cell, popular for any consumer-grade electronic: from remotes to mouses, from clocks to thermometers.  This one can't be recharged",
     "ascii_picture": "light_disposable_cell",
-    "delete": { "flags": [ "RECHARGE" ] },
     "capacity": 16,
     "pocket_data": [ { "pocket_type": "MAGAZINE", "rigid": true, "ammo_restriction": { "battery": 16 } } ]
   },
+  {
+    "id": "light_cell_rechargeable",
+    "copy-from": "light_battery_cell",
+    "type": "MAGAZINE",
+    "name": { "str": "light battery (rechargeable)", "str_pl": "light batteries (rechargeable)" },
+    "//": "AA, Nickel-Metal Hydride, ENERGIZER NH15-2300",
+    "weight": "27 g",
+    "description": "A light AA battery cell, popular for any consumer-grade electronic: from remotes to mouses, from clocks to thermometers.  This one can't be recharged",
```

Comment:
Typo in description, it says it can't be recharged, but it's rechargable battery